lib/sinks/sqlInjection/pg_client.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const pg = require('pg');
+const hooker = require('hooker');
+const escape = require('escape-html');
+
+// Prevent the query method from requiring a real database connection.
+hooker.hook(require('pg/lib/client').prototype, 'query', {
```

Comment:
Can we access `pg/lib/client` from the `pg` object required above? is it different from `pg.Client.prototype`?